src/mapper/ObsMapper.js

```diff
@@ -37,6 +37,10 @@ export class ObsMapper {
       }
       if (!obs.voided) {
         obs.voided = obs.value.indexOf('voided') > 0;
+        if (obs.voided) {
+          // removing voided prefix for Images/Videos (voided obs) before sending to bahmni apps
```

Comment:
@manimaarans removing voided **suffix**

Reply:
updated the comment @sivareddytw . Thanks